operators/transformation/mergemap.md

```diff
@@ -92,6 +106,23 @@ const example = source.mergeMap(
 const subscribe = example.subscribe(val => console.log(val));
 ```
 
+##### Example 5: mergeMap memory leak
+
+( [jsFiddle](https://jsfiddle.net/ElHuy/31y8cv1h/) )
+
+```js
+//Non-stopping source
+const source = Rx.Observable.interval(100);
+
+//Dangerous remapping
+const example = source.mergeMap(val => Rx.Observable.interval(50));
```

Comment:
I think this is good but can we think of a more realistic scenario? Maybe something from clicks that map to an inner observable? An example of it not getting cleaned up correctly + how to fix?

Reply:
I think I have an idea in mind involving user interaction.